vite.config.ts

```diff
@@ -32,6 +32,7 @@ export default defineConfig({
   },
   optimizeDeps: {
     esbuildOptions: {
+      target: 'es2020',
```

Comment:
This was necessary because of errors I received in the logs:

<details><summary>See traceback</summary>

```
[0] ✘ [ERROR] Big integer literals are not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari13" + 2 overrides)
[0]
[0]     node_modules/@alephium/web3/dist/src/constants.js:24:19:
[0]       24 │ exports.ONE_ALPH = 10n ** 18n;
[0]          ╵                    ~~~
[0]
[0] ✘ [ERROR] Big integer literals are not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari13" + 2 overrides)
[0]
[0]     node_modules/@alephium/web3/dist/src/constants.js:24:26:
[0]       24 │ exports.ONE_ALPH = 10n ** 18n;
[0]          ╵                           ~~~
[0]
[0] ✘ [ERROR] Big integer literals are not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari13" + 2 overrides)
[0]
[0]     node_modules/@alephium/web3/dist/src/constants.js:25:22:
[0]       25 │ exports.DUST_AMOUNT = 10n ** 15n;
[0]          ╵                       ~~~
[0]
[0] ✘ [ERROR] Big integer literals are not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari13" + 2 overrides)
[0]
[0]     node_modules/@alephium/web3/dist/src/constants.js:25:29:
[0]       25 │ exports.DUST_AMOUNT = 10n ** 15n;
[0]          ╵                              ~~~
[0]
```

</details>